### PR TITLE
Upgrade to Rails 4, remove CMSimple, prep for new layout

### PR DESCRIPTION
**Pull request by jayzes**
This subsumes #2 and will get merged into #4.

/cc @marlabrizel

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 
-ruby '2.1.5'
+ruby '2.3.0'
 
-gem 'rails', '~> 3.2.19'
+gem 'rails', '~> 4.2.6'
 gem 'rake'
 gem 'pg'
 
+gem 'rails4_upgrade'
```
Comment (marlabrizel) [inline Gemfile]: I added this in to help but we can probably pull it out now.

File: Gemfile [hunk 2]
```diff
@@ -52,14 +49,14 @@ gem 'active_model_serializers', '~> 0.8.0'
...
+gem 'rollbar'
 gem 'newrelic_rpm'
 
 gem 'rack-canonical-host'
 
 # Admin interface
-gem 'activeadmin'
-gem 'meta_search',    '>= 1.1.0.pre'
+gem 'activeadmin', '>=1.0.0.pre2'
+# gem 'meta_search',   '>= 1.1.0.pre'
```
Comment (jayzes) [inline Gemfile]: May need to make sure this is still working?

File: Gemfile [hunk 3]
```diff
@@ -74,17 +71,17 @@ group :assets do
   gem 'coffee-rails'
   gem 'underscore-rails'
   gem 'uglifier'
-  gem 'utensils', github: 'modeset/utensils'
-  gem 'turbo-sprockets-rails3', github: 'ndbroadbent/turbo-sprockets-rails3'
+  # gem 'utensils', github: 'modeset/utensils'
+  # gem 'turbo-sprockets-rails3', github: 'ndbroadbent/turbo-sprockets-rails3'
```
Comment (jayzes) [inline Gemfile]: Think these can stay out for now, but may need to cherry-pick in later.

File: app/models/submission.rb [hunk 4]
```diff
@@ -4,39 +4,28 @@ class Submission < ActiveRecord::Base
...
-                  :is_public,
-                  :is_confirmed,
-                  :venue_id,
-                  :budget_needed,
-                  :volunteers_needed,
-                  :start_hour,
-                  :end_hour,
-                  :state,
-                  :submitter_id, as: :admin
+  # Add to ActiveAdmin as strong params
```
Comment (marlabrizel) [inline app/models/submission.rb]: I left these commented out with a note as I believe ActiveAdmin is still broken.

File: config/initializers/cookies_serializer.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Be sure to restart your server when you modify this file.
+
+Rails.application.config.action_dispatch.cookies_serializer = :json
```
Comment (jayzes) [inline config/initializers/cookies_serializer.rb]: May want to make this `hybrid`

**Comment by marlabrizel** [comment]
Thanks for finishing this up.

This looks good to me, but we should note that ActiveAdmin is still broken. I can make a separate issue to track.

**Comment by marlabrizel** [comment]
closes #1

**Pull request merged**